packages/sdk/contractkit/src/wrappers/SortedOracles.ts

```diff
@@ -46,53 +48,87 @@ export interface MedianRate {
   rate: BigNumber
 }
 
+export interface PairWithIdentifier {
+  pair: string
+  identifier: Address
+}
```

Comment:
I couldn't find any usages of `pair` aside from the return statement in `newPairWithIdentifier` (that already takes the pair as a parameter), so I'm curious on how useful this field is?